xml/ay_create_control_file.xml

```diff
@@ -187,6 +187,34 @@
     describes the syntax and classes of an &ay; profile.
    </para>
 
+   <note>
+    <title>Schema Extensions</title>
+
+    <para>
+     &ay; can be extended by other products and modules, but the schema does not
+     contain the specification for those extensions. As a consequence, &ay;
+     a profile that use one of those extensions might be reported as invalid.
```

Comment:
"&ay; a profile" is missing something, does this mean "when &ay; is given a profile that uses one of those extentions, it might report the profile as invalid"?

Reply:
Yes, you are right. Thanks for fixing it.